src/component-library/TextLink/TextLink.style.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+import { resolveTextColor } from '../Text';
+import { Colors } from '../utils/prop-types';
+
+type BaseTextLinkProps = {
+  $color?: Colors;
+};
+
+const BaseTextLink = styled.a<BaseTextLinkProps>`
+  color: ${({ $color }) => resolveTextColor($color)};
+
+  &[aria-disabled='true'] {
+    cursor: default;
+    pointer-events: none;
+    opacity: 50%;
```

Comment:
See above comment, but if we do want to keep this style has this been agreed with UX?